js/script.js

```diff
@@ -0,0 +1,108 @@
+"use strict";
+
+// Global object to store currency rates
+const currencyRates = {
+  timestamp: Date.now(),
+  base: "EUR",
+  date: new Date().toISOString().split("T")[0],
+  rates: {},
+};
+
+// Function to insert a new currency rate (Base and Target)
+const insertRate = () => {
+  const baseCurrency = document
+    .getElementById("baseCurrency")
+    .value.toUpperCase();
+  const targetCurrency = document
+    .getElementById("targetCurrency")
+    .value.toUpperCase();
```

Comment:
Good idea! Also, use `trim` to avoid having currencies made of spaces like this:
<img width="273" alt="image" src="https://github.com/user-attachments/assets/06cb65de-06f1-42b3-a614-3b4bef7fa678">